src/main/java/uk/gov/pay/adminusers/persistence/entity/StripeAgreementEntity.java

```diff
@@ -37,6 +39,14 @@ public StripeAgreementEntity(int serviceId, String ipAddress, LocalDateTime agre
         this.ipAddress = ipAddress;
         this.agreementTime = agreementTime;
     }
+    
+    public static StripeAgreementEntity from(StripeAgreement s) {
```

Comment:
ha I battled with this one, for brevity and keep it on one line I went with `s` since its as simple method, but since you've raised it that sways me, I'll change it :)